go-controller/pkg/cni/types/types.go

```diff
@@ -2,14 +2,21 @@ package types
 
 import (
 	"github.com/containernetworking/cni/pkg/types"
-	"net"
 )
 
 // NetConf is CNI NetConf with DeviceID
 type NetConf struct {
 	types.NetConf
 	// PciAddrs in case of using sriov
 	DeviceID string `json:"deviceID,omitempty"`
+	// Network Cidr
+	NetCidr string `json:"netCidr,omitempty"`
+	// Network MTU
+	MTU int `json:"mtu,omitempty"`
+	// captures net-attach-def name in the form of namespace/name
+	NadName string `json:"netAttachDefName,omitempty"`
+	// set true if it is not the default net-attach-def
```

Comment:
"the default net-attach-def" is unclear to me. Is that 'default network interface', other than 'secondary using net-attach-def"? If so, NadName could distinguish between default network and secondary network (i.e. NadName == "" must be secondary).